apps/web/src/pages/Home/HomePage/components/Hero/Hero.tsx

```diff
@@ -49,11 +57,12 @@ const Hero = ({ defaultImage }: HeroProps) => {
   const intl = useIntl();
   const paths = useRoutes();
   const homeMobile = landscapeRandomize(defaultImage).mobile;
+  const homeTablet = landscapeRandomize(defaultImage).tablet;
```

Comment:
Right now your image will suddenly change if you resize the page.  You should probably only call this once and use the properties of a single returned value.  Perhaps even use useState to preserve through rerenders.  Also, consider renaming this function now that it returns three sizes.


https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/1f2f78ca-7e5a-4496-b7c8-94cda491eb98

Reply:
@petertgiles So there's definitely something wonky going on with the inspector on first load here - it's loading the default `srcset` image instead of the proper one for the rendered fake resolution. As far as I can tell, this doesn't happen when you simply shrink your window down to mobile size and reload. 

I've gone ahead and updated the function to only call the single random photo - good catch!

Reply:
Looks good!